compat-tool/compat.py

```diff
@@ -6,8 +6,8 @@
 import re
 import argparse
 
-
-versions = ['3.6','4.0','5.0','EC5.0']
+# FerretDB major-minor versions
+versions = ['1.0', '1.1', '1.2', '1.3', '1.4']
```

Comment:
I wonder if we could add to that array instead of replacing it. We could use `'FerretDB-1.4'`, for example. Or maybe even just `'FerretDB'` for now, because I'm not sure we want to track version right now